database/datatypes.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from datetime import datetime
+from typing import TypedDict
```

Comment:
TypeDict is only available for the latest version and I had python3.8 and it was working fine.

but now I ran from different environment 
OS : Ubuntu
Python 3.6 
and TypeDict is not included.
@ChakshuGupta : I suppose we don't want to just support python3.8 and support other version which are greater than 3.5 so we have to find a different solution.